app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,19 @@
+class CommentsController < ApplicationController
+
+  def create
+    @company = Company.find(params[:company_id])
+    @comment = Comment.new(comment_params)
+    @comment.job_id = params[:job_id]
+
+    @comment.save
```

Comment:
be sure to handle the errors so a user has another chance to create the comment if they don't create it correctly the first time 👍